xbmc/music/MusicDatabase.cpp

```diff
@@ -12062,7 +12054,7 @@ void CMusicDatabase::ExportToXML(const CLibExportSettings& settings,
                 }
               }
               xmlDoc.Clear();
-              TiXmlDeclaration decl("1.0", "UTF-8", "yes");
+              // TiXmlDeclaration decl("1.0", "UTF-8", "yes");
```

Comment:
same here

Reply:
Same reason, a reminder of what type of thing `decl` is and how it is initialised